exo-setup/features/setup.feature

```diff
@@ -20,14 +21,77 @@ Feature: Setup of Exosphere applications
       | dashboard/Dockerfile     |
       | mongo-service/Dockerfile |
       | web-server/Dockerfile    |
+    And it has generated the file "docker-compose.yml" with the content:
+      """
+      version: '3'
```

Comment:
Are we checking this file into Github? If yes, then it should have a comment on top saying:

> This file is auto-generated. Please don't modify it.

If no, it shouldn't be generated in this directory. We can still generate it in a temp directory, but shouldn't bother the user with it.